app/renderer/actions/Inspector.js

```diff
@@ -587,13 +587,14 @@ export function callClientMethod (params) {
   return async (dispatch, getState) => {
     console.log(`Calling client method with params:`); // eslint-disable-line no-console
     console.log(params); // eslint-disable-line no-console
-    const {driver, appMode} = getState().inspector;
+    const {driver, appMode, enableMobileSauceCmd} = getState().inspector;
```

Comment:
i assume this is a typo and should be `enableMobileSourceCmd`? otherwise i don't see what this has to do with sauce labs?